frontend/packages/portal-frontend/src/compound/heatmapTab/HeatmapTab.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useCallback, useState } from "react";
+import React, { useCallback, useState } from "react";
```

Comment:
Ignore this file. The changes were all previously reviewed in https://github.com/broadinstitute/depmap-portal/pull/346